testing/tests/DevExpress.ui.widgets/popup.tests.js

```diff
@@ -2859,18 +2869,20 @@ QUnit.module('positioning', {
             assert.strictEqual(args.event.target, this.$resizeHandle.get(0), 'event parameter target is correct');
         });
 
-        QUnit.test('fullScrren change after drag should trigger visualPositionChanged event with correct parameters', function(assert) {
+        QUnit.test('fullScreen change after drag should trigger visualPositionChanged event with correct parameters', function(assert) {
             const visualPositionChangedStub = sinon.stub();
             this.popup.on('visualPositionChanged', visualPositionChangedStub);
 
+            const position = this.getPosition();
+
             this.drag();
-            const { left, top } = this.getPosition();
 
             this.popup.option('fullScreen', true);
             assert.deepEqual(visualPositionChangedStub.getCall(1).args[0].position, { top: 0, left: 0 }, 'position parameter is correct after change to true');
 
             this.popup.option('fullScreen', false);
-            assert.deepEqual(visualPositionChangedStub.getCall(2).args[0].position, { top, left }, 'position parameter is correct after change to false');
+
+            assert.deepEqual(visualPositionChangedStub.getCall(2).args[0].position, position, 'position parameter is correct after change to false');
```

Comment:
Let's concretise the case here
```suggestion
            assert.deepEqual(visualPositionChangedStub.getCall(2).args[0].position, position, 'position is equal to the initial position before drag');
```